operator/controller/src/main/resources/application.properties

```diff
@@ -1,4 +1,43 @@
-quarkus.operator-sdk.crd.validate = true
-quarkus.operator-sdk.crd.apply = false
-quarkus.operator-sdk.crd.generate = false
+# Configuration env. variable format:
+# APICURIO_{group}_{name...}_{optional suffix to indicate expected value, e.g. SECONDS or ENABLED}
+
+# Logging
+
+quarkus.log.level=info
+%dev.quarkus.log.level=debug
+%test.quarkus.log.level=debug
+
+apicurio.log.level=info
```

Comment:
nitpick: I think `apicurio` configurations have no effect in this subproject.

Reply:
There are loggers in the project in the `io.apicurio` scope.